draft-ietf-quic-recovery.md

```diff
@@ -79,13 +79,40 @@ recovery, and where applicable, attributes the TCP equivalent in RFCs,
 Internet-drafts, academic papers, and/or TCP implementations.
 
 
-## Notational Conventions
+# Conventions and Definitions
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
 document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
+Definitions of terms that are used in this document:
+
+Sender:
+
+: The endpoint sending QUIC packets.
+
+Receiver:
+
+: The endpoint receiving QUIC packets and sending acknowledgements in
+  response.
+
```

Comment:
I don't think we need the above ones. In fact they may be more confusing than useful (sender of acks, for instance)